akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -22,8 +22,9 @@ class Member private[cluster] (
   val status:                    MemberStatus,
   val roles:                     Set[String]) extends Serializable {
 
-  lazy val team: String = roles.find(_.startsWith("team-"))
+  lazy val team: String = roles.find(_.startsWith(ClusterSettings.TeamRolePrefix))
     .getOrElse(throw new IllegalStateException("Team undefined, should not be possible"))
```

Comment:
I created a separate ticket for rolling upgrade considerations: https://github.com/akka/akka/issues/23243